### PR TITLE
Speak like Yoda

### LINKED ISSUE #980: Speak-Like-Yoda
# Aim

- A python script that translates an input English sentence into Yoda-speak.

# Details

- In this project, The script only uses Python's standard modules random and string, so no additional installation is needed
- @kaustubhgupta please have a look into this issue because I think this is a unique project and its something new.

### PR DESCRIPTION
**Pull request by soumik2012**
# Description

Add  the files to the folder speak to yoda folder 

## Fixes  #980

Replace `issue_no` with the issue number which is fixed in this PR

## Have you read the [Contributing Guidelines on Pull Requests](https://github.com/avinashkranjan/Amazing-Python-Scripts/blob/master/CONTRIBUTING.md)?

- [x] Yes
- [ ] No

## Type of change

- [x] New feature (non-breaking change which adds functionality)

## Checklist:

- [x] My code follows the style guidelines(Clean Code) of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have created a helpful and easy to understand `README.md`
- [x] My documentation follows [`Template for README.md`](https://github.com/avinashkranjan/Amazing-Python-Scripts/blob/master/Template%20for%20README.md)
- [x] My changes generate no new warnings
- [x] I have added tests/screenshots(if any) that prove my fix is effective or that my feature works.

# Project Metadata
  If there is no-file/nothing to fill the below fields with, then type: none

Example: If no requirements.txt needed/present then type in Requirments none => Requirments: none

Category:
- [ ] Calculators
- [ ] AI/ML
- [ ] Scrappers
- [ ] Social_Media
- [ ] PDF
- [ ] Image_Processing
- [ ] Video_Processing
- [ ] Games
- [ ] Networking
- [ ] OS_Utilities
- [ ] Automation
- [ ] Cryptography
- [ ] Computer_Vision
- [x] Fun
- [ ] Other

Title: Speak like Yoda 

Folder: Speak like Yoda 

Requirements: none

Script: Speak_Like_yoda.py

Arguments: none

Contributor: soumik2012 

Description: A python script that translates an input English sentence into Yoda-speak.

### CODE REVIEW

**Comment by soumik2012** [comment]
@kaustubhgupta please check my PR

File: Speak-like-Yoda/requirements.txt [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# Speak-Like-Yoda
```
Comment (kaustubhgupta) [inline Speak-like-Yoda/requirements.txt:1]: why requirements file contain readme text?

File: Speak-like-Yoda/Speak_Like_yoda.py [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+        sentence = sentence.replace(p, '')
+    words = sentence.split()
+    random.shuffle(words) 
+    new_sent = ' '.join(words)
+    print()
+    print('Your Yodenglish sentence:')
+    print(new_sent.capitalize())
+    
+if __name__ == '__main__':
+    print('Your English sentence:')
```
Comment (kaustubhgupta) [inline Speak-like-Yoda/Speak_Like_yoda.py:22]:
```suggestion
    print('Your English sentence: ')
```

File: Speak-like-Yoda/Speak_Like_yoda.py [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+    :param sentence: input string
+    :return: translation to Yoda-speak
+    """
+    sentence = sentence.lower()
+    for p in string.punctuation.replace("'", ''):
+        sentence = sentence.replace(p, '')
+    words = sentence.split()
+    random.shuffle(words) 
+    new_sent = ' '.join(words)
+    print()
```
Comment (kaustubhgupta) [inline Speak-like-Yoda/Speak_Like_yoda.py:17]: why empty print statement?

File: Speak-like-Yoda/Speak_Like_yoda.py [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+    print('Your Yodenglish sentence:')
```
Comment (kaustubhgupta) [inline Speak-like-Yoda/Speak_Like_yoda.py:18]:
```suggestion
    print('Your Yodenglish sentence: ')
```

**Review by kaustubhgupta — CHANGES_REQUESTED** [review]

File: Speak-like-Yoda/requirements.txt (hunk 1 above)
Comment (soumik2012) [reply Speak-like-Yoda/requirements.txt]: @kaustubhgupta by mistake I wrote on the requirement.txt

File: Speak-like-Yoda/Speak_Like_yoda.py (hunk 3 above)
Comment (soumik2012) [reply Speak-like-Yoda/Speak_Like_yoda.py:17]:
```
Your English sentence:
Avoiding failure is something we learn at some later point in life.
                      #this line for print()
Your Yodenglish sentence:
Something is we point some failure learn avoiding later at in life
```

**Comment by soumik2012** [comment]
Please check my PR @kaustubhgupta

File: Speak-like-Yoda/Speak_Like_yoda.py (hunk 3 above)
Comment (kaustubhgupta) [reply Speak-like-Yoda/Speak_Like_yoda.py:17]: Use `\n` in starting of  `print('Your Yodenglish sentence: ')`

**Review by kaustubhgupta — CHANGES_REQUESTED** [review]

**Comment by soumik2012** [comment]
@kaustubhgupta done!

**Review by kaustubhgupta — APPROVED** [review]

**Review by antrikshmisri — APPROVED** [review]

**Comment by soumik2012** [comment]
@santushtisharma10 please check my PR

**Review by santushtisharma10 — APPROVED** [review]

**Comment by soumik2012** [comment]
@kaustubhgupta please check for further more modification

**Review by Kushal997-das — APPROVED** [review]
LGTM 💯

**Review by kaustubhgupta — CHANGES_REQUESTED** [review]
@soumik2012  Update the PR template with project metadata. Look at this PR for reference: https://github.com/avinashkranjan/Amazing-Python-Scripts/pull/1002

**Review by kaustubhgupta — APPROVED** [review]

**Comment by soumik2012** [comment]
@kaustubhgupta any problem with my code I am asking because my code is not merged yet?

**Comment by kaustubhgupta** [comment]
> @kaustubhgupta any problem with my code I am asking because my code is not merged yet?

Actually, the problem is we are working on automation that adds your contribution data directly to the website (yet to be released) and in the master database. We are facing some issues with that and when it's fixed, all the pending PRs will be merged

**Comment by soumik2012** [comment]
@kaustubhgupta why the bot added the stale sir

**Comment by kaustubhgupta** [comment]
> @kaustubhgupta why the bot added the stale sir

Bot has gone mad 😆

**Comment by soumik2012** [comment]
@kaustubhgupta 🤣🤣 that why i am asking sir

**Pull request merged**